### PR TITLE
feature/AOS2-47-open-source-dashboard-unit-test-api-service

### PR DESCRIPTION
**Pull request by vdtdev**
- Pull endpoint URLs out into enum
- Update and add JSDoc comments for each method in service
- Reach 100% test coverage

### CODE REVIEW

**Review by m-thompson-code — CHANGES_REQUESTED** [review]
Looks good to me. Thanks for updating the jdocs. Only one code change suggestion. Lemme know your thoughts / if you can move to models directory

**Review by m-thompson-code — APPROVED** [review]
+1

**Comment by m-thompson-code** [comment]
Gonna wait for @krivanek06 to approve then once we've both approved, feel free to merge when you want to @vdtdev

**Review by krivanek06 — APPROVED** [review]

**Pull request merged**
